tools/calculator.html

```diff
@@ -3,38 +3,46 @@
 <head>
 <meta charset="UTF-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<title>Calculator</title>
+<title>JOHN'S Calculator</title>
+  <link 
+  rel="icon"
+  href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAOEAAADhCAMAAAAJbSJIAAAA0lBMVEVUbnpgfYv8iQL////l5eVRa3aJm6NEW2Q7WmlmfIexglGDmqX/pDRog5D8jxW4g1pXfY5/kprVdAJNZG7JbQJHYW1LZ3T4hwL2fgDxgQCmrrKQmZ7whAvYbAD45NfYiEbY3N61vMD38OvidADsxaqVpay5wcXk6OqeqKxicnvv8fLT2t3Bys5/k5tyh5FcdYDFYwBzgYiPf25+mqjJn3XpvZ5Uc4LwuJDegi/wkzDuo1v9hgDsdwDgghbfpXTu2MXNeyPUkFDPfi+cb0zMdRPZml/1yI9HAAAFLElEQVR4nO3d/V+jNgDH8XgpLWxzSluo1nqDU8RSaOu26jZvp+e5//9fWugTj3fLlAST+35+5AfL2wSaoK+W0LqiWewHXYUKAj+eRbUWUj0U+kHS7SZJYitUkp5y4Iccwpjx7Lm1IMRQKUIW1txmyPg/hDH7TcwZjqgYO+s5G8r4G8IwSGzLUJO3yTAsOwnCrwnjLvO1fY6vjhkLw5gTjtP5qUXzZFwjjIJkof4AbjIWSRCVhVFgt31ejWbviTthoMsM3TUPisKxbkBGHOeFsa3LJZhl2HEmDJO2T0dIdrgXBlbbJyOkRbATzg71m6NpxuFsK+zqCWTE7kY402CpVp9hzdbCQFcgIwapMNRmsVbNWIRM6OsLZESfCTV8s88ybEqiRdtnIbRFRGZu2ychNHdGfM2FPtH4vSLNCIjWN5r0VkP0XHRnWUTvWynR3ocQQgghhFAD1e/sNNrvuXbdAwj3RptH2M515DuVo67V8TQZRue60+lUiAzY0YS4BlaIayAjTtV/gLYFlojujbc56in/twh33OlUidsR1ILoZMKMmAN2OlPFhYZZJbo32SFvWr3LKpbRKxNdy9MJuB7FKCOapSmqAbA8Uc0c0DO0AJYnqtAparh8Nf66eaJQoD3mq3miWUMUMEWd3CX/raJe4y9dJQq5izp1c0WOsDJRxbxNtCksjaKg98FWhcSZ5l6CvS8KeY02hYU3+pr9YiO1Kcwv1cQRW7yXlkZQFNG9OeOr8deuAYohDhy+mt6xVaaowInaSvkR9KZ1W2LFK0xRw6nZEitecQSd2l2/0lV39NVdv9IZNTt6Roz0ITqzbAT326X8RI1uFH/W5g52xPxiO0c8E7IIltlgSyzuJvbX4llP8SEkKdHLXYO7tk/gzkz1gelKyqvZD64nqh7AdBTjmg0vm6jXmgDTURzUHDWaXwAjhBBCCCGEEEIIIYQQQgghhBBSpLY/EF94pCc5U3bEktvBrz9Ijvwot99+/0ly5FBux0fvfpYbOZDb8dH9O7lBCCGEEEIIIYQQQgghhBBCCCGEEEIIIYQQQgghhBBCCGGrQjvZpK2QdLZfl2zrKnS9rXAOIYQQaiscTfg6VVZ4Qfm6PIEQQgghhPB7Fn7gatWc8LZQJiwev21KeHrO18WoKWG3+PH51+FW6Jc+V5+DyLdqG3HG8aO4hLdXnNeF25SwySCEEEIIIZQk7F75haKtKC4evuL4YW9TeHBb/Eogc7+mKX1VEMeP4hOe8MX1y3qT69LT93zdKbzy5kvlvQWEEEIIIYSvFU5+4euPxp5iyBaOTjnjAL5NYaNB+J0Ltf1fjFs7WH/NWmDpKjz4H8+4FRW+IgghhBBCCCGEEEIBjbQXTsgLdkBKCf8kXb2F93+RsebCOxK/ZA+kjvDkkoQcf7hVWPjxPaELrYV/U0IDqdNUsvD+ExPOpE5TycKLByakL3iepIzwiaZCX+a6Rq7w42otpANthUO6EfoSr0SpwvPVVkin8m6nMoWnj3Qn9BwdhaPlw15Ix9KIEoXndzQT0rksojxh/zPNC+lU0t1GmrD/SIvCiMghyhL2nz+UhDSaSpmocoSj4eMOmAnZtWhKWNxIEV4sP2esnJCOewPhWykJwkl/eUfrhdSbmoZgo3Ah8z0+0K8J2QJuIHgcxQpHk+FyuCqSSkJmnJq9ARGmPD4ShRtNLvrL5dOqDKoIKZ0FxOz1nIGQ79Nwv/TFNFwul8+fHqqcGiErjMe2ZQwE5HwZiuj56Z/V5ftay7/Up7gM4kjGTQAAAABJRU5ErkJggg==">
```

Comment:
Here you are actually defining the PNG content, can you move it to a separate PNG file in icons directory and import from there?